src/main/java/dev/notebook/app/notebook/service/NotebookService.java

```diff
@@ -0,0 +1,16 @@
+package dev.notebook.app.notebook.service;
+
+import dev.notebook.app.notebook.entity.Notebook;
+import java.util.List;
+
+public interface NotebookService {
+
+    Notebook saveNotebook(Notebook notebook);
+
+    List<Notebook> getAllNotebooks();
+
+    void deleteNotebookById(Long id);
+
+    Notebook getNotebookById(Long id);
```

Comment:
```suggestion
    Notebook get(Long id);
```
В тебе і так сервіс називається NotebookService. Очевидно, що ти оперуєш ноутбуками тут